### PR TITLE
[16.0][IMP] account_invoice_mass_sending: missing commits from 15

### PR DESCRIPTION
**Pull request by sergio-teruel**
Recover missing commits from v15

https://github.com/OCA/account-invoicing/commit/faabd09b330eabf4b177d2d2a791789806fd99bf
https://github.com/OCA/account-invoicing/commit/6f1b6740a37069597b8c1afe565848900d3d1a5f

cc @Tecnativa

ping @chienandalu @CarlosRoca13

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: account_invoice_mass_sending/README.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ This module adds a mass sending feature on invoices, using asynchronous jobs.
 Configuration
 =============
 
-queue_job module needs to be configured, see the module `documentation <https://github.com/OCA/queue/tree/14.0/queue_job>`_
+queue_job module needs to be configured, see the module `documentation <https://github.com/OCA/queue/tree/15.0/queue_job>`_
```
Comment (pedrobaeza) [inline account_invoice_mass_sending/README.rst]: This README seems outdated.

Comment (sergio-teruel) [reply account_invoice_mass_sending/README.rst]: Changes Done!!

**Review by jguenat — APPROVED** [review]
I'm not against the change but could you explain why the definition of the job channel and function is necessary?
It is my understanding that it has no effect on its own (when only one channel) and also depends on how you define your channels capacity in odoo.conf.

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
AFAIK, the idea is to be able to categorize jobs for being able to dedicate numbers for each channel.

/ocabot merge minor

**Pull request merged**